tests/http/test_http_parser.py

```diff
@@ -24,10 +24,10 @@ def setUp(self) -> None:
         self.parser = HttpParser(httpParserTypes.REQUEST_PARSER)
 
     def test_issue_127(self) -> None:
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaises(ValueError):
```

Comment:
These CMs like `pytest.raises()`/`pytext.warns()` usually accept the second argument for the message. It is the best practice to always pass that for precise exception expectations.

Reply:
Agreed, this needs to be covered across all `raise` expectations I guess.